src/main/resources/i18n/messages.properties

```diff
@@ -119,10 +119,10 @@ FoldTab.64=<html><strong>Default:</strong> Enabled if filename prettifying is en
 FoldTab.65=Monitor played status of files
 FoldTab.66=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> Folders must have "Monitor played status of files" enabled to have their contents included in the New Media folder.</html>
 FoldTab.67=Do nothing
-FoldTab.68=Add an overlay to the thumbnail
-FoldTab.69=Hide the file
-FoldTab.70=Move the file to a different folder
-FoldTab.71=Move the file to the recycle/trash bin
+FoldTab.68=Mark media
+FoldTab.69=Hide video
+FoldTab.70=Move file to a different folder
+FoldTab.71=Move file to recycle/trash bin
 FoldTab.72=Fully played action:
```

Comment:
I think we're talking about slightly different things here, so I don't think so. I thought about "file" and found out that to be exact, that's what we handle in the two latter cases (we move or recycle the files themselves). In the first case, we mark any media type inside UMS (but we don't do anything to the files), and in the second case we only hide video media. I didn't think it was necessary to spell out "video media", but if you feel that is more consistant it's fine for me..? But I think the separation between what happens inside UMS (media) and what happens on a file system level (file) is relevant here.